readme.md

```diff
@@ -59,3 +85,7 @@ pnpm changeset
 ### Running tests
 
 See <app/laboratory/tests/README.md>
+
+
+> [!NOTE]
+> Looking for Web3modal v[1-5]? [switch the branch](https://github.com/WalletConnect/web3modal/tree/V5).
```

Comment:
I don't think we should guide them to switch anything about w3m, let them know how to migrate to AppKit instead https://docs.reown.com/appkit/upgrade/to-reown-appkit-web#migrate-from-web3modal-v5-to-reown-appkit